1_fetch/src/get_nwis_data.R

```diff
@@ -109,47 +109,36 @@ get_peak_flow_log <- function(files_in, file_out) {
 prescreen_daily_data <- function(filename, prov_rm = TRUE){
   #loads data from file and removes provisional and estimated data if prov_rm = TRUE
   message('loading ', filename)
-  ##Handle sites with strange column names
-  if (length(grep(pattern = '01011500', filename)) +
-      length(grep(pattern = '02196000', filename)) + 
-      length(grep(pattern = '03213000', filename)) + 
-      length(grep(pattern = '12010000', filename)) > 0){
+  d0 <- read_csv(filename,
+                 name_repair = "minimal", 
+                 col_types = cols(agency_cd = col_character(), 
+                                  site_no = col_character(), Date = col_date(format = "%Y-%m-%d"), 
+                                  discharge = col_number(), discharge_cd = col_character(), 
+                                  dateTime = col_character(), tz_cd = col_character())) %>%
```

Comment:
See comments below. I think we can just load the data here, and remove NAs in the next step to avoid losing data
```suggestion
                                  dateTime = col_character(), tz_cd = col_character()))
```

Reply:
Changed.